--- sdl/opengl.c	Fri Aug 10 10:15:18 2018
+++ sdl/opengl.c	Thu Sep 20 21:13:19 2018
@@ -225,7 +225,8 @@
 
 	// create an OpenGL compatibility context, not a core or ES context
 #ifndef WIN // except on Windows, where some Nvidia drivers really don't like us doing this
-	SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_COMPATIBILITY);
+//	zanac, these attributes don't work well in allwinner (rasbperry too?)
+//	SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_COMPATIBILITY);
 #endif
 
 	// get window and initialize OpenGL context
@@ -235,8 +236,9 @@
 		printf("Failed to create OpenGL-compatible window (%s)...", SDL_GetError());
 		goto error;
 	}
-	if((context = SDL_GL_GetCurrentContext()))
-		SDL_GL_DeleteContext(context);
+//	zanac, it seems that you should always create a new context without delete the old one, this happen in allwinner (rasbperry too?)
+//	if((context = SDL_GL_GetCurrentContext()))
+//		SDL_GL_DeleteContext(context);
 	context = SDL_GL_CreateContext(window);
 
 	// make sure the context was created successfully
