--- Makefile.orig	2020-04-14 08:57:10.689851617 +0000
+++ Makefile	2020-04-14 08:58:48.326929241 +0000
@@ -139,16 +139,20 @@
 TARGET_PLATFORM = PANDORA
 BUILD_LINUX     = 1
 BUILD_SDL       = 1
+BUILD_GLES      = 1
+BUILD_OPENGL    = 1
+BUILD_LOADGL    = 1
+BUILD_WEBM      = 1
 BUILD_GFX       = 1
 BUILD_PTHREAD   = 1
 BUILD_SDL_IO    = 1
 BUILD_TREMOR    = 1
 BUILDING        = 1
-CC  	        = $(PNDDEV)/bin/arm-none-linux-gnueabi-gcc
+CC  	        = gcc
 INCLUDES        = $(PNDDEV)/include \
                   $(PNDDEV)/include/SDL
 OBJTYPE         = elf
-LIBRARIES       = $(PNDDEV)/lib
+LIBRARIES       = .
 ifeq ($(BUILD_PANDORA), 0)
 BUILD_DEBUG     = 1
 endif
